core/templates/components/ratings/rating-display/rating-display.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for rating display component.
+ */
+
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { RatingDisplayComponent } from './rating-display.component';
+
+describe('Rating display component', () => {
+  let componentInstance: RatingDisplayComponent;
+  let fixture: ComponentFixture<RatingDisplayComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [],
+      declarations: [
+        RatingDisplayComponent
+      ],
+      providers: [],
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RatingDisplayComponent);
+    componentInstance = fixture.componentInstance;
+  });
+
+  it('should initialize', () => {
+    spyOn(componentInstance, 'displayValue');
+    componentInstance.ngOnInit();
+    expect(componentInstance.status).toEqual(componentInstance.STATUS_INACTIVE);
+  });
```

Comment:
Please separate the content in the unit tests into the following pre-check, setup, action, post-checks whichever is applicable by adding a new line. Please do this for all the unit tests.

Reply:
done